src/util.rs

```diff
@@ -0,0 +1,14 @@
+use log::warn;
```

Comment:
is this file referenced in more then one file?

Reply:
dropped